jina/serve/runtimes/gateway/grpc/__init__.py

```diff
@@ -48,6 +48,19 @@ async def async_setup(self):
             options=[
                 ('grpc.max_send_message_length', -1),
                 ('grpc.max_receive_message_length', -1),
+                # for the following see this blog post for the choice of default value https://cs.mcgill.ca/~mxia3/2019/02/23/Using-gRPC-in-Production/
+                ('grpc.keepalive_time_ms', 10000),
+                # send keepalive ping every 10 second, default is 2 hours.
+                ('grpc.keepalive_timeout_ms', 5000),
```

Comment:
may it be too short? This could be a problem with dalle flow perhaps with so slow processing time? should we maybe expose it as parameter?

Reply:
I exposed it as parameters now as well